src/core/parcel-validation.ts

```diff
@@ -0,0 +1,146 @@
+import { generateLazyValidator, JSONSchema, ValidateFunction } from '../validation'
+import { isOutOfBoundsParcel } from './parcel-exceptions'
+import { KeywordDefinition } from 'ajv'
+
+/**
+ * Represents a parcel with X and Y coordinates
+ * @public
+ */
+export type Parcel = { x: number; y: number }
+
+/**
+ * Standard limits for parcels in Decentraland
+ * @public
+ */
+export const PARCEL_LIMITS = {
+  minX: -150,
+  maxX: 150,
+  minY: -150,
+  maxY: 150
+}
+
+/**
+ * Namespace with utilities to validate parcels
+ * @public
+ */
+export namespace Parcel {
+  /**
+   * Custom validation to check if coordinates are within limits
+   */
+  export const _isInLimits: KeywordDefinition = {
+    keyword: '_isInLimits',
+    validate: function validate(schema: boolean, data: Parcel) {
+      if (!data || typeof data.x !== 'number' || typeof data.y !== 'number') {
+        return false
+      }
+
+      const { x, y } = data
+      // Check if it's within the standard limits
+      const inStandardLimits =
```

Comment:
We have the function `isInStandardBounds` which can be used here, what do you think about using it?